clients/python-wrapper/lakefs/object.py

```diff
@@ -539,7 +537,22 @@ def path(self) -> str:
 
     def reader(self, mode: ReadModes = 'rb', pre_sign: Optional[bool] = None) -> ObjectReader:
         """
-        Context manager which provide a file-descriptor like object that allow reading the given object
+        Context manager which provide a file-descriptor like object that allow reading the given object.
+
+        Usage Example:
+
+        .. code-block:: python
+
+            import lakefs
+
+            obj = lakefs.repository("<repository_name>").branch("<branch_name>").object("file.txt")
+            file_size = obj.stat().size_bytes
+
+            with obj.reader(mode='r', pre_sign=True) as fd:
+                # print every other 10 chars
+                while fd.tell() < file_size
+                    print(fd.read(10))
+                    fd.seek(10)
```

Comment:
seek(10) will not cause to read the same 10 bytes all the time? the default for seek is abs location.

Reply:
Fixed, thanks